src/api.js

```diff
@@ -68,7 +68,13 @@ export async function getScreenings(query = "") {
     return payload.data;
   }
 }
-
+export async function getAverageRating(movieId) {
+  const res = await fetch(
+    APIData + "/reviews?populate=movie&filters[movie]=" + movieId
+  );
+  const content = await res.json();
+  const reviews = content.data;
+=======
```

Comment:
It looks like you accidentally made some mistakes in handling merge conflicts when you merged with `main`. On line 77 is a merge marker, and the functions on lines 71 and 78 seem to be the same function, of different versions.

This is an easy mistake to make, so it's important to check your diffs and try running the program before committing a merge and pushing it.